_data/education.yml

```diff
@@ -223,6 +223,16 @@ websites:
       img: mystudylife.png
       tfa: No
 
+    - name: Pennsylvania State University
+      url: http://www.psu.edu/
+      img: psu.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      software: Yes
+      hardware: Yes
+      doc: http://www.identity.psu.edu/services/authentication-services/two-factor/self-service-portal/
```

Comment:
I think we should change the URL to either their overall service topic on 2FA (http://www.identity.psu.edu/services/authentication-services/two-factor/) which links to several articles, including the one about enabling it, or their 2FA enrollment site (http://get2fa.psu.edu/).

Reply:
I say use the First Link since it is more general.

Reply:
@Sunit-Kulkarni I'm hopping in this thread late, but based on the comments from review it sounds like once you update that URL this PR should be good to go

Reply:
Alrighty. Just updated it!